docs/source/science/GEDI/GEDI_L4B.ipynb

```diff
@@ -2,15 +2,15 @@
  "cells": [
```

Comment:
<p>The previous code showed how to find the collection shortname. We should either drop this section (and link to an example of How to find a Collection name), or revert back.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>The previous way didn't work for some reason, not really sure why but it wouldn't pull that collection even though the short name was correct. There was also something wrong with pprint and it wouldn't print it, so that's why we reverted to this way of pulling with the short name</p>